plugins/module_utils/template.py

```diff
@@ -207,3 +207,14 @@ def get_l3out_node_routing_policy_object(self, uuid=None, name=None, fail_module
                 "L3Out Node Routing Policy", existing_l3out_node_routing_policy, [KVPair("uuid", uuid) if uuid else KVPair("name", name)], fail_module
             )
         return existing_l3out_node_routing_policy  # Query all objects
+
+    def get_interface_policy_group_uuid(self, interface_policy_group):
+        """
+        Get the UUID of an Interface Policy Group by name.
+        :param interface_policy_group: Name of the Interface Policy Group to search for -> Str
+        :return: UUID of the Interface Policy Group. -> Str
+        """
+        existing_vlan_pools = self.template.get("fabricPolicyTemplate", {}).get("template", {}).get("interfacePolicyGroups", [])
```

Comment:
existing_vlan_pools -> existing_policy_groups

Reply:
will change it, thanks!